unit_tests/src/test_ticket_administration.cpp

```diff
@@ -0,0 +1,365 @@
+#include <catch2/catch.hpp>
+
+#include "irods/connection_pool.hpp"
+#include "irods/rodsClient.h"
+#include "irods/ticket_administration.hpp"
+#include "irods/query_builder.hpp"
+#include "irods/user_administration.hpp"
+
+#include <fmt/format.h>
+
+#include <string>
+#include <iostream>
+#include <stdio.h>
+#include <stdexcept>
+
+TEST_CASE("ticket administration")
+{
+    namespace tic = irods::experimental::administration::ticket;
+    std::string read_ticket_name = "testTicket";
+    std::string write_ticket_name = "testTicket2";
```

Comment:
Consider changing the string values.
- `"testTicket"` -> `"readTicket"`
- `"testTicket2"` -> `"writeTicket"`